components/RegisterForm.js

```diff
@@ -0,0 +1,286 @@
+import styled from "styled-components";
+import {useState} from "react";
+
+export default function RegisterForm({
+  loginMode,
+  onClickParent,
+  onClickChild,
+  showPassword,
+  onShowPassword,
+}) {
+  const [inputPassword, setInputPassword] = useState("");
+  const [inputRepeatPassword, setInputRepeatPassword] = useState("");
+
+  const handleChangePassword = event => {
+    setInputPassword(event.target.value);
+    console.log(inputPassword);
+  };
```

Comment:
Please remove all `console.log` before creating a pull request. A production version should not produce debug output.